src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializerExt.cs

```diff
@@ -22,7 +22,7 @@ public static bool CanWriteMetric(Metric metric)
         return true;
     }
 
-    public static int WriteMetric(byte[] buffer, int cursor, Metric metric, PrometheusMetric prometheusMetric, bool openMetricsRequested = false)
+    public static int WriteMetric(byte[] buffer, int cursor, Metric metric, PrometheusMetric prometheusMetric, bool openMetricsRequested = false, bool scopeInfoEnabled = true)
```

Comment:
I'm concerned about us keep adding more and more parameters to these functions, consider refactoring the code before it's getting too complex.